app.js

```diff
@@ -23,6 +26,10 @@ const cli = new CLIEngine(
 
 const report = cli.executeOnFiles(rc.files || ['.']);
 
+if (quiet) {
+  report.results = CLIEngine.getErrorResults(report.results);
```

Comment:
this came from here: https://github.com/eslint/eslint/blob/219aecb78bc646d44bad27dc775a9b3d3dc58232/lib/cli.js#L216